CMakeLists.txt

```diff
@@ -21,7 +22,7 @@ set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET 10.11)
 
-option(WITH_CXX11ABI "Use the C++11 ABI" OFF)
+option(WITH_CXX11ABI "Use the C++11 ABI" ON)
```

Comment:
Why did you change the default?

Reply:
It's been long since our development machines and CI have departed from previous C++ implementations using the old ABI.